src/test/resources/json/cucumber-v2-with-screenshot.json

```diff
@@ -0,0 +1,129 @@
+[
```

Comment:
instead of adding new test json file add change you want to test into sample.json file

Reply:
I've removed the version specific json files and added an embedding with the new media type to the existing `sample.json`.